docs/quickstart.md

```
@@ -147,7 +147,7 @@ If new commits are pushed after planning, run `.plan to dev` again before applyi
 
 !!! warning "Apply the saved plan"
 
-    Normal apply uses the saved plan for the same environment and commit. It must not run a fresh untargeted Terraform apply.
+    Normal apply uses the latest successful saved plan for the same environment and commit. It must not run a fresh untargeted Terraform apply.
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)